### DISCUSSION TITLE
Test fails after installation

### DISCUSSION THREAD

**Discussion by mishkat1096** [neperfepx/neper#194] [Q&A] [answered]
Hi,
I installed neper on WSL. After installation when I run `make test`  it encounters the following errors. FYI: `libscotch` is installed in my system

![error](https://user-images.githubusercontent.com/48925117/115271857-c793da80-a10b-11eb-945e-d94749207d8c.png)

**Comment by rquey** [comment, marked as answer]
I think that multithreading should be disable on WSL. Try to run `export OMP_NUM_THREADS=1` before running the tests.

**Reply by mishkat1096** [reply]
Thanks, it's working
